apps/web/middleware.ts

```diff
@@ -172,13 +196,11 @@ export const config = {
     "/routing-forms/:path*",
     "/team/:path*",
     "/org/:path*",
-    "/:user/:type/",
-    "/:user/",
   ],
```

Comment:
this change belongs to #19345, right?

Reply:
yup! this needs to be merged after #19345 is merged. I will draft this PR actually